### PR TITLE
refactor: added Proctored exam link and refactoring

### PR DESCRIPTION
**Pull request by PKulkoRaccoonGang**
[YouTrack](https://youtrack.raccoongang.com/issue/2U-225)

**Tasks**
- add Proctoring exam url
- Some refactoring

### CODE REVIEW

**Review by Thposbsmt — APPROVED** [review]

File: src/advanced-settings/AdvancedSettings.test.jsx [hunk 1]
```diff
@@ -61,7 +61,10 @@ describe('<AdvancedSettings />', () => {
     const { getByText } = render(<RootWrapper />);
     await waitFor(() => {
       expect(getByText(messages.headingSubtitle.defaultMessage)).toBeInTheDocument();
-      expect(getByText(messages.headingTitle.defaultMessage)).toBeInTheDocument();
+      const advancedSettingsElement = getByText('Advanced settings', {
```
Comment (ruzniaievdm) [inline src/advanced-settings/AdvancedSettings.test.jsx:64]: Why did you remove `messages.headingTitle.defaultMessage`?

Comment (PKulkoRaccoonGang) [reply src/advanced-settings/AdvancedSettings.test.jsx]: It`s my mistake, thanks
Good catch 👍

**Review by ruzniaievdm — APPROVED** [review]

File: src/generic/help-sidebar/index.jsx [hunk 2]
```diff
@@ -42,6 +42,10 @@ const HelpSidebar = ({
     `/group_configurations/${courseId}`,
     getConfig().STUDIO_BASE_URL,
   );
+  const proctoredExamSettingsDestination = new URL(
```
Comment (sendr) [inline src/generic/help-sidebar/index.jsx:45]: why didn't you use `getPagePath` function?
